app/components/user-table.js

```diff
@@ -1,2 +1,33 @@
 import Component from '@ember/component';
-export default Component.extend({});
\ No newline at end of file
+import { computed } from '@ember/object';
+export default Component.extend({
+  areIdsEqual: computed('currentTransaction', function() {
+    let currentBookFromTransaction = this.get('currentTransaction.book_id');
+    let currentBook = this.get('book.id');
+    if (currentBookFromTransaction === currentBook) {
+      return true;
+    }
+  }),
+  showCorrectBook: computed('isBookShowing', 'choosedBook', function() {
+    let isBookShowing = this.get('isBookShowing');
+    let currentBook = this.get('book.id');
+    let choosedBook = this.get('choosedBook');
+    if (isBookShowing && choosedBook == currentBook) {
+      return true;
+    }
+  }),
+  actions: {
+    showBook: function(bookId) {
+      const isBookShowing = this.get('isBookShowing');
+      if (isBookShowing) {
```

Comment:
could just say if (this.get('isBookShowing')) no real point to set it to a constant.